Makefile

```diff
@@ -131,7 +131,7 @@ go-build: $(BUILD_DIRS)
 	    -w /src                                                 \
 	    -v $$(pwd)/.go/bin/$(OS)_$(ARCH):/go/bin                \
 	    -v $$(pwd)/.go/bin/$(OS)_$(ARCH):/go/bin/$(OS)_$(ARCH)  \
-	    -v $$(pwd)/.go/cache:/.cache                            \
+	    -v $$(pwd)/.go/cache:/root/.cache/go-build              \
```

Comment:
Why `/root/.cache/go-build` and not `/root/.cache/` ?

Reply:
It seems that `/root/.cache` makes more sense, I'll push another commit, thx.